app/brave_generated_resources.grd

```diff
@@ -1351,6 +1351,12 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       </if>
       <!-- Brave VPN -->
       <if expr="enable_brave_vpn">
+        <message name="IDS_SETTINGS_SHOW_VPN_BUTTON" desc="The label for showing vpn button in the toolbar">
+          Show VPN button
+        </message>
+        <message name="IDS_SETTINGS_SHOW_VPN_BUTTON_SUB_LABEL" desc="The sub label for vpn button setting">
+            Shows the VPN button in the toolbar
```

Comment:
Is there an official text to show for these labels?

Reply:
Copied from the screenshot - https://github.com/brave/brave-browser/issues/17244#issuecomment-910460363